Simplified/Settings/NYPLSettingsDeleteServerDataViewController.swift

```diff
@@ -0,0 +1,171 @@
+    let message = NSLocalizedString("Please wait...",
+                                    comment: "Loading view message")
+    let vc = NYPLActivityIndicatorMessageViewController(with: message)
+    present(vc, animated: false)
+    NYPLAnnotations.updateServerSyncSetting(toEnabled: false) { [weak self] success in
+      NYPLMainThreadRun.asyncIfNeeded {
+        guard let self = self else {
+          return
+        }
+        self.dismiss(animated: false)
+        if success {
+          self.delegate?.didDeleteServerData()
+        } else {
+          self.showAlert()
+        }
+      }
+    }
+    DispatchQueue.main.asyncAfter(deadline: .now() + 3) {
+      self.dismiss(animated: false)
+      self.showAlert()
+    }
+  }
+  
+  @objc private func dismissVC() {
+    self.navigationController?.dismiss(animated: true, completion: nil)
+  }
+  
+  private func showAlert() {
+    let title = NSLocalizedString("Delete Data Failed",
+                                  comment: "Title for deletion failure alert")
+    let message = NSLocalizedString("CheckConnection",
+                                    comment: "Message for deletion failure alert")
+    let alert = NYPLAlertUtils.alert(title: title, message: message)
+    NYPLAlertUtils.presentFromViewControllerOrNil(alertController: alert,
+                                                  viewController: self,
+                                                  animated: true,
+                                                  completion: nil)
+  }
+  
+  // MARK: - Setup
+  
+  private func setupNavBar() {
+    self.title = NSLocalizedString("Delete my SimplyE Data",
+                                   comment: "Title for navigation bar")
+    
+    let backButton = UIBarButtonItem(title: NSLocalizedString("Cancel",
+                                                              comment: "Button title for dismissing view controller"),
+                                     style: .plain,
+                                     target: self,
+                                     action: #selector(dismissVC))
+    
+    self.navigationItem.leftBarButtonItem = backButton
+  }
+  
+  private func setupUI() {
+    self.view.backgroundColor = NYPLConfiguration.primaryBackgroundColor
+    self.tableView.separatorStyle = .none
+    self.tableView.allowsSelection = false
+  }
+  
+  // MARK: - Helper
+  
+  lazy var footerView: UIView = {
+    let view = UIView()
+    view.backgroundColor = .clear
+    return view
+  }()
+  
+  lazy var deleteDataButton: UIButton = {
+    let button = UIButton()
+    button.addTarget(self, action: #selector(deleteData), for: .touchUpInside)
+    return button
+  }()
+  
+  private func dataDeletionDescription() -> NSMutableAttributedString {
+    let paragraphStyle = NSMutableParagraphStyle()
+    paragraphStyle.lineSpacing = 1.25
+    paragraphStyle.lineHeightMultiple = 1.25
+    
+    let description = NSLocalizedString("DeleteServerDataDescription",
```

Comment:
if you switch to Spanish, is the English translation displayed? I'm afraid "DeleteServerDataDescription" will be displayed instead.... if so you might need to copy the English wording into Spanish and German, sadly.